### DISCUSSION TITLE
Heading 6

### DISCUSSION THREAD

**Discussion by jeffreylauwers** [nl-design-system/backlog#322] [Component suggestions]
<img width="720" alt="visuele weergave van de heading 6 component" src="https://github.com/user-attachments/assets/adefa188-4694-4a45-af82-becad43d5064">

## Naam

Heading 6

## Doel

Koptekst die in de koppenstructuur op het zesde niveau staat.

## Ook bekend als

Heading, Titel

## Gerelateerde componenten

[Heading](https://github.com/orgs/nl-design-system/discussions/220), [Heading 1](https://github.com/orgs/nl-design-system/discussions/317), [Heading 2](https://github.com/orgs/nl-design-system/discussions/318), [Heading 3](https://github.com/orgs/nl-design-system/discussions/319), [Heading 4](https://github.com/orgs/nl-design-system/discussions/320), [Heading 5](https://github.com/orgs/nl-design-system/discussions/321), [Rich Text Content](https://github.com/orgs/nl-design-system/discussions/289)

## Onderzoek

**Comment by Yolijn** [comment]
## ✨ Dit component is nu Community ✨

Help je mee hem door de Candidate stappen te krijgen?
[Heading 6 op het Candidate bord](https://github.com/orgs/nl-design-system/projects/32/views/1?filterQuery=heading+6)
